src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Truck is working");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Truck stopped");
```

Comment:
Similarly, the message 'Truck stopped' should be more informative. Reflect the class name in the message to indicate which Machine has stopped working.